terraform/modules/task-definitions/signon/main.tf

```diff
@@ -71,9 +89,25 @@ module "task_definition" {
         },
       ],
       "secrets" : [
+        {
+          "name" : "SECRET_KEY_BASE",
+          "valueFrom" : data.aws_secretsmanager_secret.secret_key_base.arn
+        },
         {
           "name" : "SENTRY_DSN",
           "valueFrom" : data.aws_secretsmanager_secret.sentry_dsn.arn
+        },
+        {
+          "name" : "DATABASE_URL",
```

Comment:
No need to fix this here but maybe add a TODO or a card - database_url shouldn't be a secret (only the password should be).

Reply:
I agree - this is something we'll need to fix in the app.

Reply:
I added a card for this https://trello.com/c/ahHLtryi/119